execute.go

```diff
@@ -17,14 +28,27 @@ func executeHelper(client *Client, method string, body []byte) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	resp, err := client.session.Do(req)
 	if err != nil {
 		return nil, err
 	}
 
-	respbody, err := io.ReadAll(resp.Body)
-	if err != nil {
-		return nil, err
+	respbody, rerr := io.ReadAll(resp.Body)
```

Comment:
And here `err` way better naming than `rerr`.

Reply:
HAHA wonder what I was thinking there :)